### DISCUSSION TITLE
Changing bucket replication

### DISCUSSION THREAD

**Discussion by ekhvalov** [seaweedfs/seaweedfs#2414] [Q&A] [answered]
Hello,
I have an s3 gateway with a bucket and bunch of volumes. If I want to replicate my bucket to another datacenter I have to change replication for every single volume via `weed shell volume.configure.replication` command.
Is there any way to change replication for all volumes of specified bucket at once?

**Comment by chrislusf** [comment, marked as answer]
Sorry, current code does not have this feature. But it should be easy to add in here. You are welcome to send a PR.

https://github.com/chrislusf/seaweedfs/blob/master/weed/shell/command_volume_configure_replication.go

**Reply by ekhvalov** [reply]
Ok, I'll send.

**Reply by ekhvalov** [reply]
Done https://github.com/chrislusf/seaweedfs/pull/2438
